mdx-layout/src/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { MDXProvider } from "@mdx-js/react";
+import * as path from "path";
+import { Playground } from "../../playground/dist/index.js";
+import { Props } from '../../props/dist/index.js';
+import { Showcases } from '../../showcases/dist/index.js';
+import { Space } from '../../space/dist/index.js';
+import "@divriots/markdown-github/dist/index.css";
```

Comment:
Because we use that in our code, maybe we should copy paste it here in this package to avoid side effects.
Also gives use the opportunity to not rely on class="markdown-body" maybe ?

Reply:
I can do it yes, but I would need at some point having a className or an element id / type to apply css.
we can do it in another PR I guess